app/components/stories/sections/SocialsSection.stories.mdx

```diff
@@ -0,0 +1,253 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs'
+import SocialsSection from '../../ch/sections/SocialsSection.vue'
+
+<Meta title="Sections/Content/Socials" component={SocialsSection} />
+
+export const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: { SocialsSection },
+  template: '<SocialsSection :section-title="sectionTitle" :section-desc="sectionDesc" :feeds="feeds" />',
+})
+
+# Socials Section
+
+Socials section is used to tease the several socials channels, with each time:
```

Comment:
Socials section is used to tease the different social channels. Each card contains the following:
- A title,
- A link,
- A hand picked post or video, or the last Tweet for Twitter feed.